### PR TITLE
Install the PostGIS Postgres extension

### PR DESCRIPTION
**Pull request by kmyers-rms**
Needed for the new rms-one-em project build

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
How was this tested before it was committed ?

**Comment by kmyers-rms** [comment]
I noticed even just the PR was actually building an image so I cancelled the build in case it tried to push the image too.

Not sure what the best way to test this would be, is there a process in place?

**Comment by kmyers-rms** [comment]
Docker doesn't like the "/" in my branch name when it tries to tag the image. Opening a different PR with a branch without special characters in it.

File: Makefile [hunk 1]
```diff
@@ -17,15 +17,15 @@ build-ubuntu-14.04-thin:
...
 	sudo docker build $(NO_CACHE) --build-arg IMAGE_TAG=buildbox \
-	-t $(PUSH_REPO)/buildbox \
+	-t $(PUSH_REPO)/buildbox:$(TAG) \
 	-f targets/ubuntu-14.04-thin/Dockerfile \
 	.
 
 push-ubuntu-14.04-thin:
-	echo 'skipping push'
-	sudo docker push ${PUSH_REPO}/buildbox
+	echo 'Pushing Docker iamge $(PUSH_REPO)/buildbox:$(TAG)'
```
Comment (ajain-rms) [inline Makefile:27]: spelling of `image` in: `'Pusing Docker iamge'`

**Comment by kmyers-rms** [comment]
@ajain-rms We ended up moving the work for this one to https://github.com/RMS/image-builder/pull/2

**Pull request closed without merging**